src/endpoints/mod_feedback.rs

```diff
@@ -112,34 +109,31 @@ pub async fn post_mod_feedback(
         .await
         .or(Err(ApiError::TransactionError))?;
 
-    Ok(HttpResponse::NoContent())
+    Ok(web::Json(ApiResponse {
```

Comment:
i think this behavior is somewhat inconsistent considering the rest of the post endpoints don't do this, but i can see why you did it. i'll probably think about it further

Reply:
ok, i'll leave it as it is rn then

Reply:
on second thought, nocontent is probably the generally the best move

Reply:
how would you get the id? through the GET endpoint?

Reply:
ye, i think if the get feedback returns the author's feedback as well, then it makes it easy to associate feedback with the id on the client (so they can delete)

i wouldn't assume people are storing the id locally anyways

Reply:
ok